main.go

```diff
@@ -90,6 +90,13 @@ func main() {
 				clicommand.MetaDataKeysCommand,
 			},
 		},
+		{
+			Name:  "oidc",
+			Usage: "Interact with Buildkite OIDC",
```

Comment:
I'm not sure, but perhaps this will help some people:

```suggestion
			Usage: "Interact with Buildkite OpenID Connect (OIDC)",
```